src/backend/joanie/settings.py

```diff
@@ -228,6 +230,16 @@ class Base(Configuration):
     }
 
     CURRENCY = values.Value(("EUR", "\N{euro sign}"), environ_name="CURRENCY")
+    MARION_DOCUMENT_ISSUER_CHOICES_CLASS = "howard.defaults.DocumentIssuerChoices"
+    VAT = values.Value(20, environ_name="VAT")
+    INVOICE_COMPANY_CONTEXT = values.Value(
```

Comment:
Maybe prefix these settings with `JOANIE_` like we did for `LMS_BACKENDS`. In this way we are able to quickly identify Joanie's settings. `CURRENCY` is also concerned.